accounts-db/src/ancient_append_vecs.rs

```diff
@@ -644,13 +663,50 @@ impl AccountsDb {
                 .collect::<Vec<_>>()
         });
 
+        let min_resulting_packed_slots = alive_bytes / u64::from(tuning.ideal_storage_size) + 1;
```

Comment:
to get ceiling. I added a comment.

Reply:
yes. I chose to not special case the == case. I could I guess `saturate_sub(1)` to alive bytes. That would give perfect math.

Reply:
ok, i handled the saturate sub case to get the math right.
so, the situation is: without this change, we would assume we would try to pack and then fail the whole thing.
now, we don't try to pack multi ref storages that will fail the whole thing. They will be saved for next time ancient pack is called. At that point, there will be newer, higher slot 'ancient' append vecs to pack that will become available to pack, making these same skipped multi ref storages relatively older. So, they'll be able to be packed on the next iteration. This whole change is designed to allow ancient pack to make progress with what can be packed and skip things that would cause the entire pack to fail and not happen. This entire packing fail could cause us to never make progress. We saw this on our test nodes.